src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -966,7 +966,7 @@ private function getCartShippingMethods(array $cart): array
 
         $this->ordersClient->executeCustomRequest($request);
 
-        return $this->responseChecker->getCollection($this->ordersClient->getLastResponse());
+        return $this->responseChecker->getCollection($this->ordersClient->getLastResponse())[0];
```

Comment:
Shouldn't `getCartShippingMethods` method return all the shipping methods data (instead of just the first one)?